packages/caraml/Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: setup
+setup:
+	pip install -r requirements.txt -r requirements.dev.txt
+
+.PHONY: version
+version:
+	$(eval VERSION=$(if $(OVERWRITE_VERSION),$(OVERWRITE_VERSION),$(shell ../../scripts/vertagen/vertagen.sh -p packages/caraml/ -y)))
+	@echo 'VERSION = "$(VERSION)"' > version.py
+	@echo "caraml-sdk version:" $(VERSION)
+
+.PHONY: lint
```

Comment:
do we plan to enforce formatting and/or show feedback about bad formatting in PR comments?
similar question about using pylint or any other took for static analysis?

Reply:
yeah, Will be in future MRs. Have another task related to setting up workflows for test and build SDK. Can be part of that.